### ISSUE TITLE
[Bug]: Icon not show, and question

### ISSUE THREAD

**Issue by Neutral9** [MissCorruption/QuickLootIE#74]
### What happened? (Actual vs Expected Behavior)

You fixed the one that the soulstone icons were not displaying but not completely, most of them are now displaying but I found one icon that is still not displaying I'm using Kome's Inventory Tweaks, also I was wondering about whether the icons from Completionist, Lotd curator, MoreHUD are displaying, can't get them to work.

### What mods for icons where you using, if any?

Kome's Inventory Tweaks

### Steps to Reproduce

Here's the ID of the stone that's not showing 43E26

### Version

3.0.0

### What Skyrim version are you using?

1.5.97

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MissCorruption** [comment]
I'd need screenshots to properly diagnose this. As for the extra icons simply having them installed should work fine, without logs or similar I can't help

**Comment by Neutral9** [comment]
![image](https://github.com/user-attachments/assets/d41af464-b2ed-4e4e-9532-f15a70c9ea0f)
all additional icons work normally except completionist all mods are the latest version, here is a screenshot of the missing soul stone icon and here is the logs 
[QuickLootIE.log](https://github.com/user-attachments/files/18156699/QuickLootIE.log)

**Comment by MissCorruption** [comment]
The log states:
```
[2024-12-16 22:17:27.701] [warning] [17968] [Interfaces.cpp:272] Failed to dispatch message to Completionist
[2024-12-16 22:17:27.701] [error] [17968] [RequestClient.h:26] Failed to connect to plugin Completionist (failed to deliver)
```
@AtomCrafty Did we update the API version for Comp?

**Comment by Neutral9** [comment]
@AtomCrafty Yes, works without problems, I reinstalled for a test, same thing

**Comment by AtomCrafty** [comment]
Okay, I had a chat with Faen and apparently the Completionist API initializes during `kDataLoaded`. That still shouldn't be an issue since QLIE _should_ load after Completionist since Q comes after C, but maybe the load order is somehow messed up on your end. I can try to check for issues if you upload your skse64 and Completionist logs.

If that is really the issue we could try to reinitialize the Completionist integration on `kPostLoadGame` or something similar to that. But this is the first time I hear of SKSE plugins loading in the wrong order.

**Comment by MissCorruption** [comment]
Using KIT v1 I cannot reproduce the missing icon. It shows as a regular soul gem for me in both QLIE and the regular inventory: 
![image](https://github.com/user-attachments/assets/d6e568c5-d91e-4a7b-b1f2-f21fa0ba5147)

**Comment by Neutral9** [comment]
I found a problem USSEP replaces the icon of this particular stone on a slightly different

**Comment by Neutral9** [comment]
@AtomCrafty I did some tests and found that if I overcome a certain number of simultaneously active dlls in the loading order, the Completionist icon stops showing, the exact number is 9, when any 8 .dll files are active, everything is normal, as soon as I connect any ninth one, the icon disappears. I have no idea what this is and how to get around it, I hope you can do it.

**Comment by MissCorruption** [comment]
Are you using an outdated Windows 10 version? Or maybe the LTSC release? There was a mistake that made it load only a certain number of DLLs. Other than that I wouldn't know what causes this and it doesn't seem to be caused by QuickLoot either.

**Comment by Neutral9** [comment]
i use win 11  23H2 update

**Comment by Neutral9** [comment]
There's no problem with the rest, I misspoke, but so far there's the problem I described in the last post

**Comment by Neutral9** [comment]
It's good to hear you did well. Happy New Year.

**Comment by AtomCrafty** [comment]
Can this be closed then?
